### DISCUSSION TITLE
Project Status

### DISCUSSION THREAD

**Discussion by userjack6880** [userjack6880/Open-DMARC-Analyzer#105] [Announcements]
The past few months, progress has ground to a halt. I got fairly busy in my professional life and just hadn't had the time to dedicate to this as much as I have needed to. I'll be focusing on answering questions here on GitHub as addressing issues as much as I can. I appreciate everyone that has given feedback and asked questions, truly. Thank you all.

**Comment by cweiske** [comment]
Thank you for building Open-DMARC-Analyzer and Open-Report-Parser! They helped me to inspect the reports that have been sitting in my mailbox for the past 3 years and gave me the confidence to finally enable DMARC quarantaining.
